docs/src/main/paradox/scala/http/routing-dsl/rejections.md

```diff
@@ -52,7 +47,11 @@ them. This is because some route structure produce several "reasons" why a reque
 
```

Comment:
The above section might have to be reworded a bit since the definition is no longer shown.